test/cli/simple-package/simple-package.out

```diff
@@ -10,11 +10,14 @@ foo/foo.rb:13: Unable to resolve constant `BardClass` https://srb.help/5002
 foo/foo.rb:14: Unable to resolve constant `UnexportedClass` https://srb.help/5002
     14 |    Project::Bar::UnexportedClass
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
-  Autocorrect: Use `-a` to autocorrect
-    foo/foo.rb:14: Replace with `Project::Bar::UnexportedClass`
-    14 |    Project::Bar::UnexportedClass
-            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
-    bar/bar.rb:14: Did you mean: `Project::Bar::UnexportedClass`?
+    bar/bar.rb:14: Do you need to `export` constant `Project::Bar::UnexportedClass`?
```

Comment:
This is a confusing UX to me. The `export` happens in `__package.rb`. Should this be the `__package.rb` location?